tests/console/enable_usb_repo.pm

```diff
@@ -20,12 +20,22 @@ use utils;
 
 sub run {
     select_console 'root-console';
-    my $repo_num = script_output 'zypper lr --uri | awk \'$0 ~ /hd:\/(\/\/)?\?device=\/dev\/disk\/by-id\/usb-/ {print $1}\'';
+    my $flavor = get_var("FLAVOR");
+    my $repo_num;
+    if ($flavor eq "Online") {
+        $repo_num = script_output 'zypper lr --uri | awk \'$0 ~ /hd:\/(\/\/)?\?device=\/dev\/disk\/by-id\/usb-/ {print $1}\'';
+    }
+    # In case of Full medium, test skips registration so all module repo URIs point to USB drive.
+    elsif ($flavor eq "Full") {
```

Comment:
Let me check if this test is executed anywhere else, as these flavors are only in SLE 15 SP2. Safe solution would be to introduce new code path for Full medium and put existing code in else branch.

Reply:
Ok, I've checked and didn't find any other job groups where test suite is executed, so we are fine. Nevertheless, I would at least update description so that it's visible that module works on SLE15 SP2 only and not on openSUSE distris either.

Reply:
@rwx788 Why do we expect that it should fail for older versions? For example 
https://openqa.suse.de/tests/1741344#step/enable_usb_repo/2   and
https://openqa.suse.de/tests/2329751#step/enable_usb_repo/4
Repo name looks like it has similar format with SLE15-SP2. How about Joaquin's comment? Should I leave just the code in elsif ?

Reply:
Not cause of the check itself, but cause of the `FLAVOR`, in SLE 12 SP5, it's `Server-DVD`, for SLE 15 SP1 it's ` Installer-DVD`. Meaning that none of the code paths here will be executed.

Reply:
Ah, ok. So what if I reverse the code in the condition, the first "if (flavor eq "Full")" and second part to just "else", so the original code will be executed for all other flavors?

Reply:
Yep, that's what I would propose, so that we don't need extra change in case decide to enable it somewhere else.